.travis.yml

```diff
@@ -35,6 +36,6 @@ script:
   - tests/travis/php-lint.sh ./library
   - tests/travis/php-lint.sh ./plugins
   - tests/travis/php-lint.sh ./themes
-  - phpunit -c phpunit.xml.dist
+  - ./vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover=coverage.clover
```

Comment:
Yeah, they're identical to what's already in core. The diff just threw me off. Nevermind!